sdk/servicebus/service-bus/test/internal/peekMessagesOptions.spec.ts

```diff
@@ -0,0 +1,83 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import {
+  createServiceBusClientForTests,
+  EntityName,
+  getRandomTestClientTypeWithNoSessions,
+  getRandomTestClientTypeWithSessions,
+  ServiceBusClientForTests,
+} from "../public/utils/testutils2.js";
+import { afterAll, afterEach, beforeAll, describe, it } from "vitest";
+import { expect } from "../public/utils/chai.js";
+import { ServiceBusSender } from "../../src/sender.js";
+import { ServiceBusReceiver } from "../../src/index.js";
+import { TestClientType, TestMessage } from "../public/utils/testUtils.js";
+
```

Comment:
I think we still need to import the types:

```suggestion
import " ../../src/experimental/index.js";
```

Reply:
I was surprised too, but I think that file was included in the same compilation, so TS already merged the interface.

Reply:
Does it mean it is available for all the TS users, not just who imports experimental?

Reply:
No, when building the service-bus package this import is not needed. Consumer of this still need the import statement.